docker-app/qfieldcloud/core/models.py

```diff
@@ -523,7 +523,7 @@ def random_string():
         first_letter = secrets.choice(string.ascii_lowercase)
         letters_and_digits = string.ascii_lowercase + string.digits
         secure_str = first_letter + "".join(
-            (secrets.choice(letters_and_digits) for i in range(15))
+            secrets.choice(letters_and_digits) for i in range(15)
```

Comment:
Check `black` compatibility.

Reply:
Black is fine with it or we would see errors in pre-commit I guess?

Reply:
Black is the previous step in the pre-commit.yml, so I though it might cause problems on the next run of black.

Reply:
But it would have failed with the CI, which is not the case, so ignore my comments.